Revert upstream commits 1c9785b and 3387ca5 since MacPorts has successfully
patched and built qtwebengine on Apple Silicon.

diff --git configure.pri configure.pri
index 8705ad9..e072961 100644
--- configure.pri
+++ configure.pri
@@ -140,10 +140,6 @@ defineTest(qtConfTest_detectPlatform) {
 }
 
 defineTest(qtConfTest_detectArch) {
-    macos:contains(QT_ARCHS, "arm64") {
-        qtLog("Apple Silicon not supported yet.")
-        return(false)
-    }
     contains(QT_ARCH, "i386")|contains(QT_ARCH, "x86_64"): return(true)
     contains(QT_ARCH, "arm")|contains(QT_ARCH, "arm64"): return(true)
     contains(QT_ARCH, "mips"): return(true)
diff --git src/buildtools/config/support.pri src/buildtools/config/support.pri
index 7a07b1b..e7f869a 100644
--- src/buildtools/config/support.pri
+++ src/buildtools/config/support.pri
@@ -110,10 +110,6 @@ defineTest(qtwebengine_checkForPlatform) {
 defineTest(qtwebengine_checkForArch) {
     module = $$1
     !qtConfig(webengine-arch-support) {
-        macos:contains(QT_ARCHS, "arm64") {
-            qtwebengine_skipBuild("$${module} can not be built for Apple Silicon yet.")
-            return(false)
-        }
         qtwebengine_skipBuild("$${module} can only be built for x86, x86-64, ARM, Aarch64, and MIPSel architectures.")
         return(false)
     }
